yaml/src/test/java/tools/jackson/dataformat/yaml/deser/NumberDeserWithYAMLTest.java

```diff
@@ -70,7 +70,9 @@ static class NestedBigDecimalHolder2784 {
     /**********************************************************************
      */
 
-    private final YAMLMapper MAPPER = newObjectMapper();
+    private final YAMLMapper MAPPER = mapperBuilder()
```

Comment:
This makes no sense -- None of `BigDecimal`, `Float` or `Double` is primitive... I'll need to see why there's failure.

Reply:
Ahhh. It's a BIG test class. I'll change this to be more targeted.